src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -46,6 +46,8 @@ class Translator extends BaseTranslator implements WarmableInterface
      */
     private $resources = array();
 
+    private $resourceFiles;
```

Comment:
I think this could be a "private $resourceFilesLoaded" boolean (see next comment).